src/richie/apps/courses/templates/courses/cms/fragment_course_breadcrumb.html

```diff
@@ -0,0 +1,20 @@
+{% if current_page.parent_page.parent_page.course%}
```

Comment:
you already have a change to fork when including the template that is more natural and doesn't involve any query so it would be easier to create a separate template `fragment_course_run _breadcrumb.html`?

Reply:
Right 👍 I'll refactor in a few minutes.

Reply:
New version is up.